### PR TITLE
New-entry-about-pip-in-Python

### PR DESCRIPTION
**Pull request by mdwiltfong**
### Description

I created the pip.md file in order to address [issue 213](https://github.com/Codecademy/docs/issues/213). This new entry covers what is PyPI, pip, as well as basic steps in installing it locally. 

### Type of Change

- [ X] Adding a new entry
- [ ] Editing an existing entry (fixing a typo, bug, issues, etc)
- [ ] Updating the documentation

### Checklist

- [X ] My entry follows the Codecademy Docs style guide
- [X ] I have performed a self-review of my own writing and code
- [X ] I have made corresponding changes to the documentation
- [X ] My changes generate no new warnings
- [X ] I have checked my entry and corrected any misspellings
- [X ] All writings are my own

### Codecademy Username

If you want your Codecademy username be displayed in your entry (feature coming in October), make sure to have your Codecademy user profile linked to your GitHub:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the GitHub field, add your GitHub profile URL.

### CODE REVIEW

**Comment by EugeneGohh** [comment]
There are some tests failing, feel free to check the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md).

**Comment by mdwiltfong** [comment]
I could use some help on discovering why this PR isn't passing the test. When running locally I get the following:

image


Which is odd because I didn't modify this file. There are a bunch of green checks above this line of each file passing this test. One of which is the pip.md file. As a result i'm not sure how to continue. 

Any guidance would be appreciated.

File: content/python/concepts/PIP/pip.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: ''
```
Comment (EugeneGohh) [inline content/python/concepts/PIP/pip.md:2]: According to the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md), `Title` is missing which caused the test to fail.

File: content/python/concepts/PIP/pip.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Description: ''
```
Comment (EugeneGohh) [inline content/python/concepts/PIP/pip.md:3]: According to the [Contribution Guide,](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) `Description` is missing which caused the test to fail.

File: content/python/concepts/PIP/pip.md (hunk 1 above)
Comment (mdwiltfong) [reply content/python/concepts/PIP/pip.md:2]: Excuse me! I left that there when troubleshooting. I will fill in these fields and pass on what I get in response

Comment (EugeneGohh) [reply content/python/concepts/PIP/pip.md]: Ah, let me try to help you pass in. I am opening it in my editor.

**Comment by mdwiltfong** [comment]
I have gone ahead and added a title and description. I had removed them originally in attempts to get a different error. Despite adding them, i'm still getting the same error from the test as per my previous [comment ](https://github.com/Codecademy/docs/pull/271#issuecomment-933253704)

**Comment by mdwiltfong** [comment]
I'm having some file path issues. As a result I need to remake this branch

**Comment by mdwiltfong** [comment]
I'm not sure what the problem was, but I made a new branch. As a result all the tests/checks are green on that [branch](https://github.com/Codecademy/docs/pull/272)

**Pull request closed without merging**